tests/cypress/integration/regTest_for_export_task/outerExportTask.js

```diff
@@ -0,0 +1,138 @@
+// Start writing your Cypress tests below!
```

Comment:
1. Please check comments from the previous file, many of them applicable here
2. A lot of code dublication, I would prefer rework it somehow to reduce it. Maybe to combine these two tests into the one.